Code/Tools/SceneAPI/SceneCore/Events/AssetImportRequest.cpp

```diff
@@ -80,8 +80,8 @@ namespace AZ
                     }
                     else
                     {
-                        AZ_TraceContext("Old manifest rule count", m_previousManifest.GetEntryCount());
-                        AZ_TraceContext("New manifest rule count", m_scene->GetManifest().GetEntryCount());
+                        AZ_TraceContext("Old manifest rule count", aznumeric_cast<int>(m_previousManifest.GetEntryCount()));
+                        AZ_TraceContext("New manifest rule count", aznumeric_cast<int>(m_scene->GetManifest().GetEntryCount()));
```

Comment:
nit: to be more precise with the type, `int` is not part of the overloaded `TraceContextScope` functions. use `int32_t` can be used instead.